diff --git a/C64DevMachine_win.yyp/scripts/scr_load_workspace_from_path/scr_load_workspace_from_path.gml b/C64DevMachine_win.yyp/scripts/scr_load_workspace_from_path/scr_load_workspace_from_path.gml
index ab2f79f..7021462 100644
--- a/C64DevMachine_win.yyp/scripts/scr_load_workspace_from_path/scr_load_workspace_from_path.gml
+++ b/C64DevMachine_win.yyp/scripts/scr_load_workspace_from_path/scr_load_workspace_from_path.gml
@@ -384,6 +384,13 @@ with (obj_c64_node) {
                 if (variable_struct_exists(_sm, "tool"))           _meta.tool           = _sm.tool;
                 if (variable_struct_exists(_sm, "text"))           _meta.text           = _sm.text;
                 if (variable_struct_exists(_sm, "byte_string"))    _meta.byte_string    = _sm.byte_string;
+                if (variable_struct_exists(_sm, "line_string"))    _meta.line_string    = _sm.line_string;
+                if (variable_struct_exists(_sm, "lines"))          _meta.lines          = _sm.lines;
+                if (variable_struct_exists(_sm, "active_type"))    _meta.active_type    = _sm.active_type;
+                if (variable_struct_exists(_sm, "ref_enabled"))    _meta.ref_enabled    = _sm.ref_enabled;
+                if (variable_struct_exists(_sm, "ref_asset_name")) _meta.ref_asset_name = _sm.ref_asset_name;
+                if (variable_struct_exists(_sm, "ref_offset_x"))   _meta.ref_offset_x   = _sm.ref_offset_x;
+                if (variable_struct_exists(_sm, "ref_offset_y"))   _meta.ref_offset_y   = _sm.ref_offset_y;
                 if (variable_struct_exists(_sm, "is_save_file"))   _meta.is_save_file   = _sm.is_save_file;
                 if (variable_struct_exists(_sm, "save_file_size")) _meta.save_file_size = _sm.save_file_size;
                 if (variable_struct_exists(_sm, "mc_mode"))        _meta.mc_mode        = _sm.mc_mode;
