diff --git a/C64DevMachine_win.yyp/objects/obj_workspace_manager/Draw_64.gml b/C64DevMachine_win.yyp/objects/obj_workspace_manager/Draw_64.gml
index ace33e3..7d251e8 100644
--- a/C64DevMachine_win.yyp/objects/obj_workspace_manager/Draw_64.gml
+++ b/C64DevMachine_win.yyp/objects/obj_workspace_manager/Draw_64.gml
@@ -2548,9 +2548,9 @@ if(global.workspace_disk_mode) {
         draw_set_color(c_red);
         _disk_txt += "OVER BY " + string(-_disk_free);
     }
-    _disk_txt += "  (OF " + string(_disk_cap) + " BLOCKS)";
+    _disk_txt += " /" + string(_disk_cap);
     if (!global.workspace_disk_exact) {
-        _disk_txt += "  EST";
+        _disk_txt += " EST";
     }
     draw_text_l(_stats_x,_usage_y,_disk_txt);
     _usage_y-=15;
